README.md

```diff
@@ -90,7 +90,7 @@ Copyright (c) Microsoft Corporation.  All rights reserved. Licensed under the MI
 
 ### Requirements
 * Android SDK 21+
-* [Chrome](https://github.com/Azure-Samples/active-directory-general-docs/blob/master/AndroidEmulator.md) 
+* [Chrome Custom Tabs](https://github.com/Azure-Samples/active-directory-general-docs/blob/master/AndroidEmulator.md) 
```

Comment:
Strictly speaking, there is [fallback logic to support users who do not have `CustomTabs` but _do_ have Chrome](https://github.com/AzureAD/microsoft-authentication-library-for-android/blob/dev/msal/src/main/java/com/microsoft/identity/client/AuthenticationActivity.java#L234-L238). I think this text is OK to leave as 'Chrome'

Reply:
Reverted.